docs/detectors/vcpkg.md

```diff
@@ -11,6 +11,10 @@ The vcpkg detector searches for `vcpkg.spdx.json` files produced by vcpkg during
 Because this detection strategy looks for the concrete files in the installed tree, it will accurately detect the precise packages used
 during this build and exclude packages optionally used on other platforms.
 
+## **Latest Updates**
```

Comment:
let's reword this a bit, since latest will become old at some point in time. I wouldn't link to the GH issue itself but rather grab the necessary information here. Let's be explicit on what version of VCPKG ecosystem is generating the manifest-info.json file, which when found what will the detector do. See [go.md](https://github.com/microsoft/component-detection/blob/main/docs/detectors/go.md) file example that does distinguish between two versions of Go, in here we have the before and after VCPKG version depending on what the customer used.